src/logic/prices/collections.ts

```diff
@@ -72,12 +84,17 @@ export function collectionsNFTsPricesQuery(filters: PriceFilters) {
     },
     AssetType.NFT
   )
-  return `query NFTPrices(
-    $expiresAt: String, 
-    $wearableCategory: String
-    $emoteCategory: String
-    $isWearableHead: Boolean
-    $isWearableAccessory: Boolean) {
+
+  const variablesBasedOnFilters = Object.entries(filters)
```

Comment:
the variables in the GraphQL query must be used somewhere in the query, so this new logic is to put them in dynamically just if they are being used after.